packages/yourloops/lib/data/blip-api.ts

```diff
@@ -132,7 +132,7 @@ class BlipApi {
     return await Promise.reject(new Error(translate('not-logged-in')))
   }
 
-  public async exportData(patient: IUser, startDate: string, endDate: string): Promise<Blob> {
+  public async exportData(patient: IUser, startDate: string, endDate: string): Promise<string> {
```

Comment:
Api has changed ? It now returns a string ?

Reply:
It seems that it was already returning a string...